<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaArvore</class>
 <widget class="QWidget" name="telaArvore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="ehCasa">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAdd">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsViewArvore">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Criar Nó:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lNome">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lNum">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Excluir Nó:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lRemov">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btRemover">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Remover</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addEx">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar Exemplos:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
